src/gui/Gui_TEST.cc

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <gtest/gtest.h>
+
+#include <ignition/msgs/stringmsg.pb.h>
+
+#include <ignition/common/Console.hh>
+#include <ignition/common/StringUtils.hh>
+#include <ignition/transport/Node.hh>
+
+#include "ignition/gazebo/gui/Gui.hh"
+#include "ignition/gazebo/test_config.hh"
+
+int g_argc = 1;
+char **g_argv = new char *[g_argc];
+
+using namespace ignition;
+using namespace gazebo;
+
+/////////////////////////////////////////////////
+TEST(GuiTest, PathManager)
```

Comment:
Is this expected to work on macOS? https://build.osrfoundation.org/job/ignition_gazebo-ci-pr_any-homebrew-amd64/3558/testReport/(root)/UNIT_Gui_TEST/test_ran/

Reply:
Good point, this is also a problem on `ign-gazebo2`. Disabled here and will also disable there.